flow/e2e/test_utils.go

```diff
@@ -317,6 +318,18 @@ func RunXminFlowWorkflow(env *testsuite.TestWorkflowEnvironment, config *protos.
 	env.ExecuteWorkflow(peerflow.XminFlowWorkflow, config, state)
 }
 
+// Execute fn in parallel with ExecuteWorkflow, returns when both complete
+func GoWorkflow(fn func(), env *testsuite.TestWorkflowEnvironment, workflowFn interface{}, args ...interface{}) {
+	wg := sync.WaitGroup{}
```

Comment:
tempted to make this a `chan struct{}` & then put a 1 minute timeout on channel being closed after ExecuteWorkflow finishes